Sources/XcodeGraphMapper/Mappers/Targets/PBXTarget+GraphMapping.swift

```diff
@@ -8,8 +8,9 @@ extension PBXTarget {
     func bundleIdentifier() throws -> String {
         if let bundleId = debugBuildSettings.string(for: .productBundleIdentifier) {
             return bundleId
+        } else {
+            return "Unknown"
```

Comment:
The bundle identifier can defined in an xcconfig in which case we don't read it. To properly fix this, we should read and parse xcconfigs to get the correct value. For now, we default to `Unknown` instead of failing when the project is correctly defined.